analyses/interaction-plots/scripts/02-plot_interactions.R

```diff
@@ -0,0 +1,82 @@
+# Create plot of co-occurence and mutual exclusivity
+#
+# JA Shapiro for ALSF - CCDL
+#
+# 2019
+#
+# Option descriptions
```

Comment:
Documentation's incomplete here and in the command line example.

Reply:
Will fix here, but does the command line example need all options?

Reply:
Normally I would say no, but in this case it's an input file and an output file.